docs/architecture/adr-076-combine-spec-repo.md

```diff
@@ -0,0 +1,97 @@
+# ADR 076: Combine Spec and Tendermint Repositories 
+
+## Changelog
+
+- 2022-02-04: Initial Draft. (@tychoish)
+
+## Status
+
+Accepted.
+
+## Context
+
+While the specification for tendermint was original in the same
+repository as the Go implementation, at some point the specification
+was split from the core tendermint repository and maintained
+separately from the implementation. While this makes sense in
+promoting a conceptual separation of specification and implementation,
+in practice this separation was a premature optimization apparently
```

Comment:
Do we have context from the original split? I'm pretty curious as to what justification was given for this in the past to make sure we're not ripping up a chesterton's fence.

Reply:
I tried to find an issue or commit logs to explain it, but I wasn't able to find a clear explanation. Even the commits at the time of the split are mostly just "do the split". If we _do_ have a doc somewhere, I agree it would be nice to link it.

Reply:
yeah, this is the main reason that I made an ADR for this even though I think this might be a bit of a process overshoot.